ui/components/app/edit-gas-fee-popover/edit-gas-item/index.scss

```diff
@@ -12,11 +12,15 @@
   width: 100%;
 
   &:hover:not([disabled]) {
-    background-color: var(--color-primary-muted);
+    background-color: var(--color-background-default-hover);
   }
 
   &--selected {
-    background-color: var(--color-background-alternative);
+    background-color: var(--color-primary-muted);
+
+    &:hover:not([disabled]) {
+      background-color: var(--color-primary-muted);
+    }
   }
 
   button.edit-gas-item--disabled[disabled] {
```

Comment:
Just readjusting some of the colors here so they're more correct

## Before

https://user-images.githubusercontent.com/8112138/202263452-667d8dbf-7c9a-4e33-becc-55cdc954169e.mov

https://user-images.githubusercontent.com/8112138/202263448-2620252e-066f-4c70-a318-78b18ed01673.mov

## After

https://user-images.githubusercontent.com/8112138/202262307-0794b123-80f1-4c20-9810-533d0830c328.mov

https://user-images.githubusercontent.com/8112138/202262299-f0c86559-67f1-448c-93ac-aa53fea59425.mov